ui/src/containers/configs/NewConfig.js

```diff
@@ -887,6 +891,24 @@ class NewConfig extends Component {
                             }}
                           />
                         </div>
+                        <div className="col-12">
```

Comment:
Can you please add a bit of margin to the top by introducing the CSS class `mt-2`?